code/modules/mob/dead/observer/observer.dm

```diff
@@ -293,7 +293,7 @@ Works together with spawning an observer, noted above.
 	if(isliving(former_mob))
 		recordable_time = former_mob.timeofdeath
 
-	ghost.client?.player_details.time_of_death = recordable_time
+	ghost.persistent_client?.time_of_death = recordable_time
```

Comment:
does this deserve `?`

Reply:
idk probably not but i dont want this code exploding on me